### ISSUE TITLE
Add a time-limited condition to SOC weblogs

### ISSUE THREAD

**Issue by marklkelly** [fastly/waflyctl#85]
# Please add a time-limited condition to SOC web-logging when provisioning a WAF

## Overview
The goal of this feature would be to add a condition to the SOC-logging web-log entry that evaluates to `false` after a given period of time has elapsed since the WAF was provisioned.

The default period of time would be 60 days, but providing an option to override this duration within the configuration file would be a requirement. This override could also serve to suppress the condition completely if a specific override value is supplied, e.g. `0`. 

The condition should be deployed by default during the provisioning of a WAF.

A simple example of what the condition might look like:
```
if (std.atoi(now.sec) < 1556129227){
  log "this message will stop being logged after Tue, 24 Apr 2019 18:07:07 GMT";
}
```

**Comment by ejthurgo** [comment]
When specifying an override in the configuration file / CLI, would specifying a number of days be enough, or would you need require accuracy?

**Comment by marklkelly** [comment]
Days is fine.

**Comment by ejthurgo** [comment]
### Usage:
Value of `0` will disable log expiry.
CLI will override config.

#### Config file snippet:
 ```
[weblog]
name = "soc-weblogs"
address = "0.0.0.0"
expiry = 5	 # expiry from current time (on local machine), expressed in days.
port = 0
 ```

#### CLI:
 ```
--web-log-expiry=90
 ```

**Comment by fgsch** [comment]
Addressed in aba80ef967598a4a0822853510bc87550099edea.
